exercises/rock_paper_scissors.livemd

```diff
@@ -10,8 +10,11 @@ Mix.install([
   {:ecto, "~> 3.7"},
   {:math, "~> 0.7.0"},
   {:faker, "~> 0.17.0"},
-  {:utils, path: "utils"}
+  {:utils, path: "utils"},
+  {:tested_cell, git: "https://github.com/BrooklinJazz/tested_cell"}
 ])
+
+Elixir.TestedCell.Control.show_editors()
```

Comment:
I don't think we should commit calls to `show_editors/1`. Ideally, students shouldn't see the editors, this is only for teachers/contributors.

Reply:
Oops sorry, forgot to remove it before committing 🤦‍♂️